ext_emconf.php

```diff
@@ -16,21 +16,16 @@
     'author' => 'Kitodo. Key to digital objects e.V.',
     'author_email' => 'contact@kitodo.org',
     'author_company' => 'http://www.kitodo.org/',
-    'shy' => '',
-    'priority' => '',
-    'module' => '',
     'state' => 'stable',
     'internal' => '',
     'uploadfolder' => TRUE,
     'createDirs' => '',
-    'modify_tables' => '',
     'clearCacheOnLoad' => FALSE,
-    'lockType' => '',
     'version' => '2.2.0',
     'constraints' => array (
         'depends' => array (
-            'php' => '5.3.7-',
-            'typo3' => '6.2.0-7.9.99',
+            'php' => '7.0.8-',
+            'typo3' => '7.6.0-',
         ),
         'conflicts' => array (
```

Comment:
If we run in composer mode, the composer.json is checked and the class loading is managed by the autoconf settings.

In non-composer mode, we need this information in ext_emconf.php. Supposed the solarium is installed below "vendor/", the following setting is working for me:

<pre>
'autoload' => array(
   'classmap' => array(
      'vendor',
      'common/class.tx_dlf_solr.php',
      'cli/class.tx_dlf_cli.php',
      'common/class.tx_dlf_alto.php',
      'common/class.tx_dlf_document.php',
      'common/class.tx_dlf_format.php',
      'common/class.tx_dlf_fulltext.php',
      'common/class.tx_dlf_helper.php',
      'common/class.tx_dlf_indexing.php',
      'common/class.tx_dlf_list.php',
      'common/class.tx_dlf_mods.php',
      'common/class.tx_dlf_module.php',
      'common/class.tx_dlf_plugin.php',
      'common/class.tx_dlf_solr.php',
      'common/class.tx_dlf_teihdr.php',
      'hooks/class.tx_dlf_doctype.php',
      'hooks/class.tx_dlf_em.php',
      'hooks/class.tx_dlf_hacks.php',
      'hooks/class.tx_dlf_tceforms.php',
      'hooks/class.tx_dlf_tcemain.php',
      'modules/indexing/index.php',
      'modules/newclient/index.php',
      'plugins/audioplayer/class.tx_dlf_audioplayer.php',
      'plugins/basket/class.tx_dlf_basket.php',
      'plugins/collection/class.tx_dlf_collection.php',
      'plugins/feeds/class.tx_dlf_feeds.php',
      'plugins/listview/class.tx_dlf_listview.php',
      'plugins/metadata/class.tx_dlf_metadata.php',
      'plugins/navigation/class.tx_dlf_navigation.php',
      'plugins/newspaper/class.tx_dlf_newspaper.php',
      'plugins/oai/class.tx_dlf_oai.php',
      'plugins/pagegrid/class.tx_dlf_pagegrid.php',
      'plugins/pageview/class.tx_dlf_pageview.php',
      'plugins/search/class.tx_dlf_search.php',
      'plugins/search/class.tx_dlf_search_suggest.php',
      'plugins/statistics/class.tx_dlf_statistics.php',
      'plugins/toc/class.tx_dlf_toc.php',
      'plugins/toolbox/class.tx_dlf_toolbox.php',
      'plugins/toolbox/tools/pdf/class.tx_dlf_toolsPdf.php',
      'plugins/toolbox/tools/fulltext/class.tx_dlf_toolsFulltext.php',
      'plugins/toolbox/tools/imagemanipulation/class.tx_dlf_toolsImagemanipulation.php',
      'plugins/toolbox/tools/imagedownload/class.tx_dlf_toolsImagedownload.php',
      'plugins/validator/class.tx_dlf_validator.php'
    )
),
</pre>

Reply:
Ok, than I add a vendor folder and some infos about manual installation to the docs?

Reply:
As noted below, not the whole vendor directory should be added, but 'vendor/solarium'.
Thank you!